src/store/modules/invites.js

```diff
@@ -13,16 +13,13 @@ export default {
     },
   },
   actions: {
-    async claim({ rootState: { current, sdk }, rootGetters: { account } }, secretKey) {
+    async claim({ rootState: { sdk }, rootGetters: { account } }, secretKey) {
       const publicKey = Crypto.getAddressFromPriv(secretKey);
-      // TODO: Remove this after fixing https://github.com/aeternity/aepp-sdk-js/issues/1261
-      const { name, instance } = sdk.pool.get(current.network);
-      // TODO: Remove this after merging https://github.com/aeternity/aepp-sdk-js/pull/1060
-      const s = await Universal({
-        nodes: [{ name, instance }],
-        accounts: [MemoryAccount({ keypair: { publicKey, secretKey } })],
-      });
-      await s.transferFunds(1, account.address, { payload: 'referral', verify: false });
+      await sdk.transferFunds(
+        1,
+        account.address,
+        { payload: 'referral', onAccount: { publicKey, secretKey }, modal: false },
```

Comment:
also for the referral would be good to be enum value

Reply:
We don't have any other hardcoded payloads. And we are not checking for this payload somewhere in the wallet. I assume this payload was set to track usage of this feature or so from outside. I think it is fine to keep it as it is, also the intention of PR was to remove the specific workaround.